src/users/users.service.ts

```diff
@@ -103,4 +105,19 @@ export class UsersService {
       throw new NotFoundException(`There isn't any user with id: ${id}`);
     }
   }
+
+  async setRoleUser(email: string, role: Role, update_data: any): Promise<any> {
```

Comment:
I'm not recommend using `any` here, we should avoid using `any` if we could and use it if and only if we have no choice. But since we just only want it to work, I would say this is LGTM.